src/audio/module_adapter/module_adapter.c

```diff
@@ -1046,10 +1046,8 @@ static int module_adapter_copy_dp_queues(struct comp_dev *dev)
 				       source_get_data_available(data_src));
 
 		err = source_to_sink_copy(data_src, data_sink, true, to_copy);
-		if (err) {
-			comp_err(dev, "LL to DP copy error status: %d", err);
-			return err;
-		}
+		if (err)
+			comp_dbg(dev, "LL to DP copy error status: %d", err);
```

Comment:
not even a warning? Is this a common scenario and really nothing to worry about?

Reply:
at this place is almost not possible as to_copy is calculated based on current free/avail